projects/arrow-java/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Google LLC
```

Comment:
nit: 2022 here and elsewhere

Reply:
Fixed. Thanks for your careful review. @asraa @jonathanmetzman